ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-security.j2

```diff
@@ -13,15 +13,32 @@ ip security
 {%         if ike_policy.local_id is arista.avd.defined %}
       local-id {{ ike_policy.local_id }}
 {%         endif %}
+{%         if ike_policy.ike_lifetime is arista.avd.defined %}
+      ike-lifetime {{ ike_policy.ike_lifetime }}
+{%         endif %}
+{%         if ike_policy.encryption is arista.avd.defined and ike_policy.encryption != "disabled" %}
+      encryption {{ ike_policy.encryption }}
+{%         endif %}
```

Comment:
Thanks for this!

the idea is still to render `encryption null` when encryption == "disabled" in the model as it will replace "null" eventually - it makes for a bulkier template but prevent the potential issue of entering `null` instead of `"null"` in ansible variables that Claus raised

Reply:
please ignore this - I was reviewing the wrong place..